spec/models/monthly_report_spec.rb

```diff
@@ -19,58 +19,27 @@
 require 'rails_helper'
 
 RSpec.describe MonthlyReport, type: :model do
-  it { expect(subject).to respond_to(:user_id) }
-  it { expect(subject).to respond_to(:project_summary) }
-  it { expect(subject).to respond_to(:used_technology) }
-  it { expect(subject).to respond_to(:responsible_business) }
-  it { expect(subject).to respond_to(:business_content) }
-  it { expect(subject).to respond_to(:looking_back) }
-  it { expect(subject).to respond_to(:next_month_goals) }
-  it { expect(subject).to respond_to(:month) }
-  it { expect(subject).to respond_to(:year) }
+  describe 'Validations' do
+    subject { build(:monthly_report) }
 
-  describe '.create' do
-    let(:report) do
-      MonthlyReport.new(
-        user_id: user_id,
-        project_summary: p_summary,
-        used_technology: u_tech,
-        responsible_business: r_busi,
-        business_content: b_cont,
-        looking_back: l_back,
-        next_month_goals: goals,
-        month: month,
-        year: year
-      )
-    end
-    let(:user_id) { 1 }
-    let(:p_summary) { 'hoge' }
-    let(:u_tech) { 'huga' }
-    let(:r_busi) { 'foo' }
-    let(:b_cont) { 'bar' }
-    let(:l_back) { 'mogu' }
-    let(:goals) { 'muga' }
-    let(:month) { 1 }
-    let(:year) { 2020 }
+    it { is_expected.to be_valid }
+    it { is_expected.to validate_presence_of(:user_id) }
+    it { is_expected.to validate_presence_of(:month) }
+    it { is_expected.to validate_inclusion_of(:month).in_range(1..12) }
+    it { is_expected.to validate_presence_of(:year) }
+    it { is_expected.to validate_numericality_of(:year).is_greater_than_or_equal_to(2000) }
+  end
 
-    context 'correct params' do
-      it 'is valid with user_id, month, year ' do
-        expect(report).to be_valid
-      end
-    end
+  describe 'Relations' do
+    let(:report) { create(:monthly_report) }
 
-    context 'incorrect params' do
-      context 'invalid if user id is string' do
-        let(:user_id) { 'one' }
-        it { expect(report).not_to be_valid }
-      end
-      context 'invalid if month is out of 1 to 12 ' do
-        let(:month) { 0 }
-        it { expect(report).not_to be_valid }
-      end
-      context 'invalid if year is under 2000' do
-        let(:year) { 1999 }
-        it { expect(report).not_to be_valid }
+    describe 'Comments' do
+      let!(:comments) { create_list(:monthly_report_comment, 3, monthly_report: report) }
+      it { expect(report.monthly_report_comments.size).to eq 3 }
+
+      context 'When report destroyed' do
+        before { report.destroy }
+        it { expect { comments.sample.reload }.to raise_error(ActiveRecord::RecordNotFound) }
```

Comment:
__[rubocop]__
* Line is too long. [93/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)